prisma/schema.prisma

```diff
@@ -59,16 +59,16 @@ model achievement_template {
   user_achievement          user_achievement[]
 
   // Text related fields
-  tagline                   String?                       @db.VarChar
-  title                     String                        @db.VarChar
-  subtitle                  String?                       @db.VarChar
-  description               String                        @db.VarChar
-  achievedDescription       String?                       @db.VarChar
-  footer                    String?                       @db.VarChar
-  achievedFooter            String?                       @db.VarChar
+  tagline             String? @db.VarChar
```

Comment:
Who reformatted it? Maybe we need to align the formatting here?

Reply:
No idea, i just executed `prisma format` as I tried to add some relations before (which are now gone again)

Reply:
Hmm, I'm noticing it's reformatting the changes I did in this PR https://github.com/corona-school/backend/pull/1081/files, then it was probably my fault.